doc/source/user_guide/ug_implementation.rst

```diff
@@ -684,18 +687,65 @@ layers and the ice thickness distribution defined by ``kcatbound`` = 0.
 Restart information for some tracers is also included in the netCDF restart
 files.
 
-Three namelist variables control model initialization, ``ice_ic``, ``runtype``,
-and ``restart``, as described in :ref:`tab-ic`. It is possible to do an
-initial run from a file **filename** in two ways: (1) set runtype =
-‘initial’, restart = true and ice\_ic = **filename**, or (2) runtype =
-‘continue’ and pointer\_file = **./restart/ice.restart\_file** where
-**./restart/ice.restart\_file** contains the line
-“./restart/[filename]". The first option is convenient when repeatedly
-starting from a given file when subsequent restart files have been
-written. With this arrangement, the tracer restart flags can be set to
-true or false, depending on whether the tracer restart data exist. With
-the second option, tracer restart flags are set to ‘continue’ for all
-active tracers.
+Three namelist variables generally control model initialization, ``runtype``,
+``ice_ic``, and ``use_restart_time``.  The valid values for ``runtype``
+are ``initial`` or ``continue``.  When ``runtype`` = `continue`, the
+initial condition is defined by the rpointer file, ``use_restart_time``
+is forced to true and ``ice_ic`` plays no role.  When ``runtype`` =
+`initial`, ``ice_ic`` has three options, ``none``, ``default``,
+or *filename*.  These initial states are no-ice, latitudinal dependent
+ice, and ice defined by a file respectively.  In `initial` mode,
+``use_restart_time`` should generally be set to false and the initial
+time is then defined by ``year_init``, ``month_init``, ``day_init``, 
+and ``sec_init``.  These combinations options are summarized in 
+:ref:`tab-ic`. 
+
+Restart files and initial condition files are generally the same file.
+They contain the model state from a particular instance in time.  In general,
+that state includes the physical and dynamical state as well as the
+state of optional tracers.  Reading of various tracer groups can
+be independently controlled by various restart flags.  In other
+words, a restart file can be used to initialize a new configuration
+where new tracers are used (i.e. bgc).  In that case, the physical
+state of the model will be read, but if bgc tracers don't exist on the
+restart file, they can be initialized from scratch.
+
+In ``continue`` mode, rpointer files are used to restart the model.
+In this mode, the CICE model writes out a small text (rpointer) file
+to the run directory that names the most recent restart file.   On
+restart, the model reads the rpointer file which defines the
+name of the restart file.  The model then reads that restart file.
+By having this feature, the ice namelist does not need to be constantly
+updated with the latest
+restart filename, and the model can be automatically resubmitted.
+Manually editing the rpointer file in the middle of a run will reset
+the restart filename, and this makes it possible to relatively easily
+backup a run to rerun.
```

Comment:
I'm not sure exactly what you mean here... also I guess you meant to write "back up" and not "backup" ?

Reply:
I have simplified that sentence.